scanners/gitleaks/scanner/Dockerfile

```diff
@@ -4,7 +4,7 @@
 
 ARG scannerVersion=latest
 FROM zricethezav/gitleaks:${scannerVersion}
-RUN adduser -S -H -u 1001 gitleaks
+USER root
```

Comment:
Why is this required?

Reply:
wrapper.sh is owned by root:root. The rights to execute this script can therefore not be changed as user 'gitleaks' that is now used in the base image